fleche/doc.ml

```diff
@@ -23,10 +23,14 @@ type node =
   ; memo_info : string
   }
 
+(* module Contents = struct *)
```

Comment:
Remove?

Reply:
Ups, sorry Ali, I think I messed up a rebase and included some stuff I should not; let me try again (after the 25th)